ruby/rbenv/files/patch-libexec-rbenv-init.diff

```diff
@@ -0,0 +1,27 @@
+This patch is needed to make
+    eval "$(rbenv init -)"
+work without complaining about
+    -bash: /opt/local/bin/../completions/rbenv.bash: No such file or directory
```

Comment:
This patch isn't actually being used by the port—you need to specify `patchfiles patch-libexec-rbenv-init.diff`.

Also I don't see the error you describe. Could you provide instructions for reproducing the issue?

Reply:
This patch was previously included. It may not be necessary any longer. I will need to check.